discord/commands/core.py

```diff
@@ -1630,14 +1639,14 @@ def validate_chat_input_name(name: Any, locale: Optional[str] = None):
         )
     if not 1 <= len(name) <= 32:
         raise ValidationError(
-            "Chat input command names and options must be 1-32 characters long. "
+            "Command names and options must be 1-32 characters long. "
             f"Received {name}" + f" in locale {locale}"
```

Comment:
```suggestion
            f"Received {name} in locale {locale}"
```
Might as well change them all if we change one...